.github/workflows/smoke-test-daily.yml

```diff
@@ -406,7 +406,7 @@ jobs:
                     - plugin: 'Gutenberg'
                       slug: gutenberg
                     - plugin: 'Gutenberg - Nightly'
-                      slug: gutenberg
```

Comment:
The slug here is used to create the report, and it was overwriting the stable gutenberg one.